app/main.py

```diff
@@ -1,3 +1,23 @@
-def shop_trip():
-    # write your code here
-    pass
+from __future__ import annotations
+import json
+from app.customer import Customer
+from app.shop import Shop
+
+
+def open_file() -> tuple[list[Customer], list[Shop], float]:
```

Comment:
> The main.py module must contain only shop_trip function

Judging by this line from the README, if you want to introduce another function, you should create a module and import it.